projects/netgrif-components/src/lib/panel/task-panel-single/single-task.component.ts

```diff
@@ -0,0 +1,23 @@
+import { Component, Inject, Optional } from '@angular/core';
+import {
+    AbstractSingleTaskComponent,
+    InjectedTabData,
+    LoggerService,
+    NAE_TAB_DATA
+} from 'netgrif-components-core';
+import { ActivatedRoute } from '@angular/router';
+
+@Component({
+  selector: 'nc-single-task',
+  templateUrl: './single-task.component.html',
+  styleUrls: ['./single-task.component.scss']
+})
+export class SingleTaskComponent extends AbstractSingleTaskComponent {
+
+  constructor(protected _log: LoggerService,
+              @Optional() @Inject(NAE_TAB_DATA) injectedTabData: InjectedTabData,
+              protected route?: ActivatedRoute) {
```

Comment:
same as before. If `route` cannot be injected Anguilar will error. `?` can be removed. The `injectedTabData` are makred as optional and the type should include `| null` since this is what will be set by Angular if the injection fails.

Reply:
Corrected.